SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -679,3 +704,21 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+void UGlobalStateManager::IncrementSessionIDAndUpdateSpatial()
+{
+	SessionId++;
+	SetSessionIDInternal(SessionId);
+}
+
+void UGlobalStateManager::SetSessionIDInternal(int32 InSessionId)
```

Comment:
I think this should be `SendSessionIdUpdate` since it's constructing and sending a component update.
Also how come this takes a `InSessionId` when we are setting the class variable `SessionId`?